From thomasr@pyramid.de Thu Sep 29 06:00:09 2005
Date: Thu, 29 Sep 2005 14:57:29 +0200 (CEST)
From: Thomas Riewe <thomasr@pyramid.de>
To: Greg Kroah-Hartman <gregkh@suse.de>
Subject: [PATCH] drivers/usb/serial/ftdi_sio: add PID/VID
Message-ID: <Pine.LNX.4.62.0509291444220.13825@amygdala>

We would like to add a PID for the Pyramid Appliance Display, which works 
on USB via FTDI_SIO.


Signed-off-by: Thomas Riewe <thomasr@pyramid.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>



 drivers/usb/serial/ftdi_sio.c |    1 +
 drivers/usb/serial/ftdi_sio.h |    3 +++
 2 files changed, 4 insertions(+)

--- gregkh-2.6.orig/drivers/usb/serial/ftdi_sio.c	2005-10-03 14:58:46.000000000 -0700
+++ gregkh-2.6/drivers/usb/serial/ftdi_sio.c	2005-10-03 14:58:54.000000000 -0700
@@ -411,6 +411,7 @@
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_UM100_PID) },
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_UR100_PID) },
 	{ USB_DEVICE(FTDI_VID, FTDI_ELV_ALC8500_PID) },
+	{ USB_DEVICE(FTDI_VID, FTDI_PYRAMID_PID) },
 	/*
 	 * These will probably use user-space drivers.  Uncomment them if
 	 * you need them or use the user-specified vendor/product module
--- gregkh-2.6.orig/drivers/usb/serial/ftdi_sio.h	2005-10-03 14:58:46.000000000 -0700
+++ gregkh-2.6/drivers/usb/serial/ftdi_sio.h	2005-10-03 14:58:54.000000000 -0700
@@ -342,6 +342,9 @@
 #define EVOLUTION_VID		0xDEEE	/* Vendor ID */
 #define EVOLUTION_ER1_PID	0x0300	/* ER1 Control Module */
 
+/* Pyramid Computer GmbH */
+#define FTDI_PYRAMID_PID	0xE6C8	/* Pyramid Appliance Display */
+
 /* Commands */
 #define FTDI_SIO_RESET 		0 /* Reset the port */
 #define FTDI_SIO_MODEM_CTRL 	1 /* Set the modem control register */
